dev/TabView/TabView.cpp

```diff
@@ -318,7 +324,10 @@ void TabView::UnhookEventsAndClearFields()
     m_addButtonColumn.set(nullptr);
     m_rightContentColumn.set(nullptr);
     m_tabContainerGrid.set(nullptr);
-    m_shadowReceiver.set(nullptr);
+    if (!SharedHelpers::Is21H1OrHigher())
+    {
```

Comment:
This check isn't needed, we can set the tracker_ref to null even if it was previously null